.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,49 @@
+name: "🆕 Feature request"
+description: "Suggest an idea for this project"
+title: "🆕 Feature request: "
+labels: [feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a new issue, please search for duplicate issues to prevent opening a duplicate feature request.
```

Comment:
IDK why, but something sounds off with this sentence ("duplicate duplicate") 😅

Reply:
Haha definitely, as the one who wrote this in the past.
Done.